plugins/google/iam/corporateEmailsOnly.js

```diff
@@ -10,12 +10,12 @@ module.exports = {
     recommended_action: 'Ensure that no users are actively using their Gmail accounts to access GCP.',
     apis: ['projects:getIamPolicy'],
 
-    run: function(cache, settings, callback) {
+    run: function (cache, settings, callback) {
```

Comment:
I think you should remove spaces around brackets here, line#18 and line#60 because we are not using spaces in these cases throughout the application so better keep it consistent.

Reply:
done